front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```diff
@@ -1,19 +1,32 @@
-.activeRow {
-  font-weight: bold;
-  background-color: var(--coolgray1);
-  --dsg-cell-background-color: var(--coolgray1);
-  --dsg-cell-disabled-background-color: var(--coolgray1);
-  .dsg-cell {
+.time-stops-datasheet {
+  .activeRow {
     font-weight: bold;
+    background-color: var(--coolgray1);
+    --dsg-cell-background-color: var(--coolgray1);
+    --dsg-cell-disabled-background-color: var(--coolgray1);
+    .dsg-cell {
+      font-weight: bold;
+    }
+  }
+  
+  .invalidCell {
+    color: red !important;
+  }
+  
+  .dsg-cell-sticky-right {
+    button {
+      margin: auto;
+    }
   }
-}
 
-.invalidCell {
-  color: red !important;
-}
+  .warning-schedule {
+    // warning-30
+    // @extend .text-warning-30;
+    background: #EAA72B !important;
+  }
 
-.dsg-cell-sticky-right {
-  button {
-    margin: auto;
+  .warning-margin {
+    // warning-30 50% alpha
```

Comment:
```suggestion
```

Reply:
we need to know what color it is from the design sheet when we have the variables